src/controllers/hid/hidcontroller.cpp

```diff
@@ -162,6 +232,7 @@ int HidController::open(const QString& resourcePath) {
         return -1;
     }
 
+#ifndef Q_OS_ANDROID
```

Comment:
In general the libusb backend of hidapi should support reading the report-descriptor. Maybe an Android permission issue?

Reply:
Unfortunately, it only works once you have allowed the app to access the device, as it requires to communicate with the device

Currently, I have made the decision to follow the given flow:
- List all device (without hid read)
- User starts a controller
- Android permission popup

In the future, we could change to:
- List all device and ask permission to get HID read
- User starts a controller


Arguably, an Android device will usually only have USB device intended to be used in Mixxx so we could assume the second flow is acceptable, but I would suggest to keep the first one for now, as there is further work to do on permission anyway